packages/compartment-mapper/src/parse-pre-cjs.js

```diff
@@ -28,37 +28,59 @@ export const parsePreCjs = async (
   );
 
   /**
-   * @param {Object} moduleExports
+   * @param {Object} moduleEnvironmentRecord
    * @param {Compartment} compartment
    * @param {Record<string, string>} resolvedImports
    */
-  const execute = async (moduleExports, compartment, resolvedImports) => {
+  const execute = (moduleEnvironmentRecord, compartment, resolvedImports) => {
     const functor = compartment.evaluate(source);
 
+    const originalExports = new Proxy(
+      Object.create(compartment.globalThis.Object.prototype),
+      {
+        get(target, prop) {
+          return moduleEnvironmentRecord[prop] || target[prop]; // this makes things like exports.hasOwnProperty() work.
```

Comment:
Not a blocker: I generally favor wrapping long comments at 80 or 120 columns, so I also consequently tend to put inline comments on the line above, to avoid a hanging indent.

Reply:
Ok. 

I've abandoned all code formatting opinions other than use a non-zero number of whitespace characters for indent working in many differently opinionated environments. Happy to adopt yours here as I go and discover them :)